Podcast/RecommendedTagsCollectionViewCell.swift

```diff
@@ -25,9 +25,10 @@ class RecommendedTagsCollectionViewCell: UICollectionViewCell {
         contentView.addSubview(tagLabel)
     }
     
-    func setupWithTag(tag: Tag) {
+    func setupWithTag(tag: Tag, fontColor: UIColor) {
```

Comment:
let's put a default value on this font color ... this is okay for now cuz its one value but in the future, if I were to make something like this I would make an enum with the types this could be used for ... and switch depending on the enum this font color and all other variables associated with the views being different

Reply:
also it's swiftier to do setup(with tag: Tag, fontColor: UIColor) :P